manageiq-providers-ansible_tower.gemspec

```diff
@@ -13,6 +13,8 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,lib}/**/*"]
 
+  s.add_runtime_dependency "ansible_tower_client", "~> 0.13.0"
```

Comment:
@Glutexo please create a new one for `0.14` as https://github.com/ansible/ansible_tower_client_ruby/commit/6e8eef437eab7b37bc72f4bbc7856850a1c88b52